### ISSUE TITLE
[Accessibility] Data grid is not accessible using narrator shortcut keys

### ISSUE THREAD

**Issue by yewmsft** [OfficeDev/office-ui-fabric-react#4083]
Screen reader shortcuts:

T = go to table
Ctrl + alt + arrows = Move around in table

These shortcuts are not working on data grid.

**Comment by oengusmacinog** [comment]
@yewmsft Can you please give us some more information about which component you are experiencing this issue on and the version of Fabric?
